lib/high_voltage/configuration.rb

```diff
@@ -25,17 +25,17 @@ def self.extended(base)
     end
 
     def action_caching=(value)
-      ActiveSupport::Deprecation.warn(CACHING_DEPRECATION_WARNING)
+      ActiveSupport::Deprecation.warn(CACHING_DEPRECATION_WARNING) if value
```

Comment:
We try to avoid trailing conditionals as much as possible and be explicit in our nil checks with a `.present?` call. Would you mind updating to the following?

```ruby
if value.present?
  ActiveSupport::Deprecation.warn(CACHING_DEPRECATION_WARNING)
end
```